nnvm/python/nnvm/top/nn.py

```diff
@@ -243,3 +243,36 @@ def schedule_upsampling(_, outs, target):
         return topi.generic.schedule_injective(outs)
 
 reg.register_pattern("upsampling", OpPattern.INJECTIVE)
+
+@reg.register_compute("lrn")
+def compute_lrn(attrs, inputs, _):
+    """Compute definition of lrn"""
+    size = attrs.get_int("size")
+    axis = attrs.get_int("axis")
+    alpha = attrs.get_float("alpha")
+    beta = attrs.get_float("beta")
+    bias = attrs.get_float("bias")
+    return topi.nn.lrn(inputs[0], size, axis, alpha, beta, bias)
+
+@reg.register_schedule("lrn")
+def schedule_lrn(attrs, outs, target):
+    """Schedule definition of lrn"""
+    with tvm.target.create(target):
+        return topi.generic.schedule_lrn(outs)
+
+reg.register_pattern("lrn", OpPattern.OUT_ELEMWISE_FUSABLE)
+
+@reg.register_compute("l2normalize")
+def compute_l2normalize(attrs, inputs, _):
+    """Compute definition of l2normalize"""
+    eps = attrs.get_float("eps")
+    axis = attrs.get_int_tuple("axis")
+    return topi.nn.l2normalize_instance(inputs[0], eps, axis)
+
+@reg.register_schedule("l2normalize")
+def schedule_l2normalize(attrs, outs, target):
+    """Schedule definition of l2normalize"""
+    with tvm.target.create(target):
+        return topi.generic.schedule_l2normalize(outs)
+
+reg.register_pattern("l2normalize", OpPattern.OUT_ELEMWISE_FUSABLE)
```

Comment:
if we want to mark it as OUT_ELEMWISE_FUSABLE, confirm with a testcase of op + elemwise operator so that it generate testcase for used ops